### PR TITLE
added underscore smybol in PRODUCT_BUNDLE_IDENTIFIER regex matching

### PR DESCRIPTION
**Pull request by imranspahic**
When creating project it can be named with underscore: example_project.
Package was not renaming `PRODUCT_BUNDLE_IDENTIFIER` for the following format: com.example.example_project.
This PR adds missing symbol in regex to correctly modify all occurrences of `PRODUCT_BUNDLE_IDENTIFIER`

### CODE REVIEW

File: lib/platforms/ios.dart [hunk 1]
```diff
@@ -126,7 +126,7 @@ void _setIOSPackageName(dynamic packageName) {
         // `PRODUCT_BUNDLE_IDENTIFIER = "{{BUNDLE_ID}}.{{EXTENSION_NAME}}";`
         .replaceAllMapped(
       RegExp(
-        r'PRODUCT_BUNDLE_IDENTIFIER = "([A-Za-z0-9.-]+)\.([A-Za-z0-9.-]+)";',
+        r'PRODUCT_BUNDLE_IDENTIFIER = ([A-Za-z0-9.-_]+)\.([A-Za-z0-9.-_]+);',
```
Comment (OutdatedGuy) [inline lib/platforms/ios.dart:129]: Why did you remove the `"` ? The extension PRODUCT_BUNDLE_IDENTIFIER have those, if removed, the regex who also match non expected elements.

**Review by OutdatedGuy — CHANGES_REQUESTED** [review]
Please resolve the requested changes

**Comment by chandrabezzo** [comment]
@OutdatedGuy I think @imranspahic misleading what I raised on the [issue](https://github.com/OutdatedGuy/package_rename/issues/48). We didn't know the functionality of this double quote

File: lib/platforms/ios.dart (hunk 1 above)
Comment (imranspahic) [reply lib/platforms/ios.dart]: @OutdatedGuy reverted back double qoutes, not sure why i have removed that. Since regex works with them as well

**Review by OutdatedGuy — APPROVED** [review]
LGTM!
